uber/templates/regform.html

```diff
@@ -18,9 +18,11 @@
         if ($.field('no_cellphone')) {
             setVisible($.field('no_cellphone').parents('.checkbox'), checked);
         }
```

Comment:
Since we bold or unbold this field depending on whether or not the volunteering checkbox is checked, we wrap that in an if-statement and leave it bolded if they're a dealer.